Failing tests due to outdated paths

## 📝 Provide detailed reproduction steps (if any)

There are some failing after our previous CKE5 bumps.

Running tests with `yarn test` doesn't work.

### ✔️ Expected result

Runner should... well, run.

### ❌ Actual result

```
$ yarn test
yarn run v1.22.17
$ ckeditor5-dev-tests -sc --files=./tests/unit/**/*.js --coverage-paths=./src
⚠ You're running tests in dev mode - some error protections are loose. Use the `--production` flag to use strictest verification methods.

START:
(...)

ERROR in ./tests/unit/dependencies/ckeditor5-markdown-gfm.js
Module not found: Error: Can't resolve '@ckeditor/ckeditor5-list/src/listediting' in 'C:\Development\htdocs\ckSource\ckeditor5\github-writer\tests\unit\dependencies'

ERROR in ./tests/unit/plugins/autoformat.js
Module not found: Error: Can't resolve '@ckeditor/ckeditor5-list/src/listediting' in 'C:\Development\htdocs\ckSource\ckeditor5\github-writer\tests\unit\plugins'

ERROR in ./tests/unit/dependencies/ckeditor5-markdown-gfm.js
Module not found: Error: Can't resolve '@ckeditor/ckeditor5-list/src/todolistediting' in 'C:\Development\htdocs\ckSource\ckeditor5\github-writer\tests\unit\dependencies'

ERROR in ./tests/unit/plugins/autoformat.js
Module not found: Error: Can't resolve '@ckeditor/ckeditor5-list/src/todolistediting' in 'C:\Development\htdocs\ckSource\ckeditor5\github-writer\tests\unit\plugins'
i ｢wdm｣: Failed to compile.
11 10 2022 22:02:13.570:INFO [karma-server]: Karma v5.2.3 server started at http://localhost:9876/
11 10 2022 22:02:13.572:INFO [launcher]: Launching browsers CHROME_LOCAL with concurrency unlimited
11 10 2022 22:02:13.652:INFO [launcher]: Starting browser Chrome
Error from chokidar (C:\): Error: EBUSY: resource busy or locked, lstat 'C:\DumpStack.log.tmp'
11 10 2022 22:02:16.126:INFO [Chrome 105.0.0.0 (Windows 10)]: Connected on socket b2UNNqNAj8bGCHkqAAAA with id 31400674
11 10 2022 22:02:18.162:WARN [reporter]: SourceMap position not found for trace: Uncaught Error: Cannot find module '@ckeditor/ckeditor5-list/src/listediting'
at http://localhost:9876/base/build/.automated-tests/entry-point.js?3d1dfd52c91dc3cc758f5266364c933d4a18b4ce:249206:150

Error: Cannot find module '@ckeditor/ckeditor5-list/src/listediting'
    at webpackMissingModule (http://localhost:9876/base/build/.automated-tests/entry-point.js?3d1dfd52c91dc3cc758f5266364c933d4a18b4ce:249206:45)
    at ./tests/unit/dependencies/ckeditor5-markdown-gfm.js (http://localhost:9876/base/build/.automated-tests/entry-point.js?3d1dfd52c91dc3cc758f5266364c933d4a18b4ce:249206:160)
    at __webpack_require__ (http://localhost:9876/base/build/.automated-tests/entry-point.js?3d1dfd52c91dc3cc758f5266364c933d4a18b4ce:20:30)
    at ./build/.automated-tests/entry-point.js (http://localhost:9876/base/build/.automated-tests/entry-point.js?3d1dfd52c91dc3cc758f5266364c933d4a18b4ce:426:161)
    at __webpack_require__ (http://localhost:9876/base/build/.automated-tests/entry-point.js?3d1dfd52c91dc3cc758f5266364c933d4a18b4ce:20:30)
    at http://localhost:9876/base/build/.automated-tests/entry-point.js?3d1dfd52c91dc3cc758f5266364c933d4a18b4ce:84:18
    at http://localhost:9876/base/build/.automated-tests/entry-point.js?3d1dfd52c91dc3cc758f5266364c933d4a18b4ce:87:10
Chrome 105.0.0.0 (Windows 10) ERROR
  Uncaught Error: Cannot find module '@ckeditor/ckeditor5-list/src/listediting'
  at webpack:///tests/unit/dependencies/ckeditor5-markdown-gfm.js:1:1 <- build/.automated-tests/entry-point.js:249206:150

  Error: Cannot find module '@ckeditor/ckeditor5-list/src/listediting'
      at webpackMissingModule (webpack:///tests/unit/dependencies/ckeditor5-markdown-gfm.js:1:1 <- build/.automated-tests/entry-point.js:249206:45)
      at ./tests/unit/dependencies/ckeditor5-markdown-gfm.js (webpack:///tests/unit/dependencies/ckeditor5-markdown-gfm.js:1:1 <- build/.automated-tests/entry-point.js:249206:160)
      at __webpack_require__ (webpack:///webpack/bootstrap:19:1 <- build/.automated-tests/entry-point.js:20:30)
      at ./build/.automated-tests/entry-point.js (webpack:///build/.automated-tests/entry-point.js:1:1 <- build/.automated-tests/entry-point.js:426:161)
      at __webpack_require__ (webpack:///webpack/bootstrap:19:1 <- build/.automated-tests/entry-point.js:20:30)
      at webpack:///webpack/bootstrap:83:1 <- build/.automated-tests/entry-point.js:84:18
      at build/.automated-tests/entry-point.js:87:10

Finished in 2.039 secs / 0 secs @ 22:02:18 GMT+0200 (Central European Summer Time)
```

### ❓ Possible solution

Upstream plugins were simply moved. It's a matter of adjusting paths.

## 📃 Other details

*   Browser: Any
*   OS: Any

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

--- a/tests/unit/dependencies/ckeditor5-markdown-gfm.js
+++ b/tests/unit/dependencies/ckeditor5-markdown-gfm.js
@@ -5,8 +5,8 @@
 
 import CodeBlockEditing from '@ckeditor/ckeditor5-code-block/src/codeblockediting';
 import KbdEditing from '@mlewand/ckeditor5-keyboard-marker/src/KbdEditing';
-import ListEditing from '@ckeditor/ckeditor5-list/src/listediting';
-import TodoListEditing from '@ckeditor/ckeditor5-list/src/todolistediting';
+import ListEditing from '@ckeditor/ckeditor5-list/src/list/listediting';
+import TodoListEditing from '@ckeditor/ckeditor5-list/src/todolist/todolistediting';
 
 import { createTestEditor } from '../../_util/ckeditor';
 import { getData } from '@ckeditor/ckeditor5-engine/src/dev-utils/model';

--- a/tests/unit/plugins/autoformat.js
+++ b/tests/unit/plugins/autoformat.js
@@ -15,8 +15,8 @@ import CodeBlockEditing from '@ckeditor/ckeditor5-code-block/src/codeblockeditin
 import HeadingCommand from '@ckeditor/ckeditor5-heading/src/headingcommand';
 import HeadingEditing from '@ckeditor/ckeditor5-heading/src/headingediting';
 import HorizontalLineEditing from '@ckeditor/ckeditor5-horizontal-line/src/horizontallineediting';
-import ListEditing from '@ckeditor/ckeditor5-list/src/listediting';
-import TodoListEditing from '@ckeditor/ckeditor5-list/src/todolistediting';
+import ListEditing from '@ckeditor/ckeditor5-list/src/list/listediting';
+import TodoListEditing from '@ckeditor/ckeditor5-list/src/todolist/todolistediting';
 
 import { setData, getData } from '@ckeditor/ckeditor5-engine/src/dev-utils/model';
 import { createTestEditor } from '../../_util/ckeditor';

